book/src/SUMMARY.md

```diff
@@ -42,6 +42,7 @@
     - [Double-and-add](design/gadgets/double-and-add.md)
     - [SHA-256](design/gadgets/sha256.md)
       - [16-bit table chip](design/gadgets/sha256/table16.md)
+    - [Endoscaling](design/gadgets/endoscaling.md)
```

Comment:
```suggestion
    - Unstable
      - [SHA-256](design/gadgets/sha256.md)
        - [16-bit table chip](design/gadgets/sha256/table16.md)
      - [Endoscaling](design/gadgets/endoscaling.md)
```
I _think_ this would make them sub-sections of an "Unstable" section in the summary, while keeping their URLs stable.